clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -303,97 +235,32 @@ private void parseContainer(XMLExtendedStreamReader reader, PathAddress subsyste
                 }
             }
         }
-
-        if (!aliases.isEmpty()) {
-            // Adapt aliases parsed from legacy schema into format expected by the current attribute parser
-            setAttribute(reader, String.join(" ", aliases), operation, CacheContainerResourceDefinition.ListAttribute.ALIASES);
-        }
     }
 
+    @SuppressWarnings("static-method")
     private void parseTransport(XMLExtendedStreamReader reader, PathAddress containerAddress, Map<PathAddress, ModelNode> operations) throws XMLStreamException {
 
         PathAddress address = containerAddress.append(JGroupsTransportResourceDefinition.PATH);
         ModelNode operation = Util.createAddOperation(address);
         operations.put(containerAddress.append(TransportResourceDefinition.WILDCARD_PATH), operation);
 
-        String stack = null;
-        String cluster = null;
-
         for (int i = 0; i < reader.getAttributeCount(); i++) {
-            String value = reader.getAttributeValue(i);
             XMLAttribute attribute = XMLAttribute.forName(reader.getAttributeLocalName(i));
             switch (attribute) {
-                case STACK: {
-                    if (this.schema.since(InfinispanSchema.VERSION_3_0)) {
-                        throw ParseUtils.unexpectedAttribute(reader, i);
-                    }
-                    stack = value;
-                    break;
-                }
-                case EXECUTOR: {
-                    if (this.schema.since(InfinispanSchema.VERSION_4_0)) {
-                        throw ParseUtils.unexpectedAttribute(reader, i);
-                    }
-                    readAttribute(reader, i, operation, JGroupsTransportResourceDefinition.ExecutorAttribute.TRANSPORT);
-                    ROOT_LOGGER.executorIgnored(JGroupsTransportResourceDefinition.ExecutorAttribute.TRANSPORT.getName());
```

Comment:
Please drop this and others from `org.jboss.as.clustering.infinispan.InfinispanLogger` (batching to this one line:)

* org.jboss.as.clustering.infinispan.InfinispanLogger#topologyAttributeDeprecated
* org.jboss.as.clustering.infinispan.InfinispanLogger#dataSourceJndiNameNotFound
* org.jboss.as.clustering.infinispan.InfinispanLogger#dataSourceNotFound
* org.jboss.as.clustering.infinispan.InfinispanLogger#indeterminiteStack
* org.jboss.as.clustering.infinispan.InfinispanLogger#executorIgnored
* org.jboss.as.clustering.infinispan.InfinispanLogger#remoteCacheCreated

Reply:
Fixed.